opencga-core/src/main/java/org/opencb/opencga/core/config/Docker.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2015-2020 OpenCB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.opencb.opencga.core.config;
+
+import org.opencb.commons.datastore.core.ObjectMap;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by jtarraga on 24/01/24.
+ */
+public class Docker {
```

Comment:
I'd have called this "DockerConfig" or something different, as "Docker" alone might be confusing